doc/admin/installation.md

```diff
@@ -1,234 +1 @@
"Preferences/{Disk, Memory}" are configured accordingly.

### Building from GitHub

To build the Docker image directly from GitHub, run the following command:

```bash
$ curl -L https://raw.githubusercontent.com/daos-stack/daos/master/utils/docker/Dockerfile.centos.7 | \
        docker build --no-cache -t daos -
```

This creates a CentOS 7 image, fetches the latest DAOS version from GitHub,
builds it, and installs it in the image.
For Ubuntu and other Linux distributions, replace Dockerfile.centos.7 with
Dockerfile.ubuntu.20.04 and the appropriate version of interest.

Once the image created, one can start a container that will eventually run
the DAOS service:

```bash
$ docker run -it -d --privileged --name server \
        -v /dev/hugepages:/dev/hugepages \
        daos
```

If Docker is being run on a non-Linux system (e.g., OSX), the export of /dev/hugepages
should be removed since it is not supported.

### Building from a Local Tree

To build from a local tree stored on the host, a volume must be created to share
the source tree with the Docker container. To do so, execute the following
command to create a docker image without checking out the DAOS source tree:

```bash
$ docker build -t daos -f utils/docker/Dockerfile.centos.7 --build-arg NOBUILD=1 .
```

Then create a container that can access the local DAOS source tree:

```bash
$ docker run -it -d --privileged --name server \
        -v ${daospath}:/home/daos/daos:Z \
        -v /dev/hugepages:/dev/hugepages \
        daos
```

${daospath} should be replaced with the full path to your DAOS source tree.
As mentioned above, the export of /dev/hugepages should be removed if the
host is not a Linux system.

Then execute the following command to build and install DAOS in the
container:

```bash
$ docker exec server scons --build-deps=yes install PREFIX=/usr
```

### Simple Docker Setup

The `daos_server_local.yml` configuration file sets up a simple local DAOS
system with a single server instance running in the container. By default, it
uses 4GB of DRAM to emulate persistent memory and 16GB of bulk storage under
/tmp. The storage size can be changed in the yaml file if necessary.

The DAOS service can be started in the docker container as follows:

```bash
$ docker exec server mkdir /var/run/daos_server
$ docker exec server daos_server start \
        -o /home/daos/daos/utils/config/examples/daos_server_local.yml
```

Once started, the DAOS server waits for the administrator to format the system.
This can be triggered in a different shell, using the following command:

```bash
$ docker exec server dmg -i storage format
```

Upon successful completion of the format, the storage engine is started, and pools
can be created using the daos admin tool (see next section).

!!! note
    Please make sure that the uio_pci_generic module is loaded.

For more advanced configurations involving SCM, SSD or a real fabric, please
refer to the next section.

```

Comment:
There's some kind of funny line-ending conversion going on in this latest revision.  I see the same thing in the `installation.md` file attached to the ticket.  I thought that was "normal" and was hoping it would not be seen in the PR review but clearly it is.

Basically the update file being pushed to GitHub has had all of it's line endings changed making it fundamentally a completely different file.  Did something change in the editing environment?